naga/src/back/glsl/mod.rs

```diff
@@ -420,24 +420,24 @@ pub struct PushConstantItem {
     ///     value: f32,
     /// }
     ///
-    /// struct PushConstant {
+    /// struct Immediate {
```

Comment:
```suggestion
    /// struct ImmediateData {
```

So that it matches the usage below.